osidb/tests/test_package_versions.py

```diff
@@ -11,30 +11,29 @@ class TestPackageVersions(object):
     def test_create_cve_v5_version(self):
         """test raw verison range creation"""
 
-        vr_1 = CVEv5Version.objects.create(
-            version="3.2.1",
+        flaw = FlawFactory()
+        pkg = PackageFactory(
+            acl_read=flaw.acl_read,
+            acl_write=flaw.acl_write,
+            flaw=flaw,  # make _validate_acl_identical_to_parent_flaw happy
         )
 
-        with pytest.raises(ValidationError):
-            vr_1.validate()
-
-        vr_2 = CVEv5Version.objects.create(
+        PackageVer.objects.create(
             version="3.2.1",
-            status=VersionStatus.UNAFFECTED,
+            package=pkg,
         )
 
-        assert vr_2.validate() is None
-        assert vr_2.save() is None
+        assert pkg.validate() is None
+        assert pkg.save() is None
 
     def test_create_cve_v5_package_versions(self):
 
-        vrs_1 = CVEv5PackageVersions()
+        vrs_1 = Package()
```

Comment:
This is most likely a useless test. A model without required fields will always raise an exception.

Reply:
Similar to discussion https://github.com/RedHatProductSecurity/osidb/pull/307#discussion_r1323017622